save-demo-cpg/src/main/kotlin/com/saveourtool/save/demo/cpg/controller/CpgController.kt

```diff
@@ -79,25 +79,26 @@ class CpgController(
                 .tap {
                     saveTranslationResult(it)
                 }
-                .map { tmpFolder.fileName.name to logs }
-                .getOrHandle { null to logs + "Exception: ${it.message} ${it.stackTraceToString()}" }
+                .map {
+                    CpgResult(
+                        getGraph(),
+                        tmpFolder.fileName.name,
+                        logs,
+                    )
+                }
+                .getOrHandle {
+                    CpgResult(
+                        getGraph(),
```

Comment:
In case of exception -- getGraph should be empty I guess

Reply:
Ok. Changed it